### PR TITLE
Completed css-media-queries exercise.

### PR DESCRIPTION
**Pull request by sharon-tieu**
https://user-images.githubusercontent.com/103021766/189064490-05560c31-261e-45a1-99a6-84cdc37fd86e.mov

https://user-images.githubusercontent.com/103021766/189064575-72559556-8de8-4a08-b6dc-24030bfced1a.mov

### CODE REVIEW

**Comment by sharon-tieu** [comment]
https://user-images.githubusercontent.com/103021766/189064686-95f0a1ff-6bab-4cca-96d4-e39aac18dc85.mov

**Comment by sharon-tieu** [comment]
https://user-images.githubusercontent.com/103021766/189067605-5aa76291-84d7-4f4e-85a1-0eb4b087e1e5.mov

File: css-media-queries/meta-viewport.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- your code here 👇 -->
+
```
Comment (BrettAlbright) [inline css-media-queries/meta-viewport.html]: make sure to include the meta viewport tag

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Comment by sharon-tieu** [comment]
Uploading Screen Recording 2022-09-08 at 10.29.41 AM.mov…

**Pull request merged**